R/estimate_KM.R

```diff
@@ -194,9 +148,18 @@ estimate_KM <- function(
       rlang::set_names(strata)
   }
 
+  # Return ------------------------------------------------------------------
+  survfit_object
+}
 
-
-# Return ------------------------------------------------------------------
-
-  return(survfit_object)
+.call_list_to_name <- function(call_list) {
```

Comment:
check overlap with the_lhs. We should only keep one.

Reply:
this little function calls `the_lhs()`. I didn't add this code, just moved it into a small helper function.

Reply:
Moved to utils file